firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/settings/network/DefaultSettingsSpiCall.java

```diff
@@ -122,7 +122,7 @@ public JSONObject invoke(SettingsRequest requestData, boolean dataCollectionToke
   /** package private for testing */
   JSONObject handleResponse(HttpResponse httpResponse) {
     final int statusCode = httpResponse.code();
-    logger.d("Settings result was: " + statusCode);
+    logger.v("Settings response code was: " + statusCode);
```

Comment:
If we're removing this from DEBUG, it is probably useful to add the response code the output below in 131. Something like: `Logger.e(Failed to retrieve settings (response=" + statusCode + ") from " + url);`

Reply:
Good call.